frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -107,77 +94,167 @@ const DashboardInfoSidebarBody = ({
     [],
   );
 
+  const canWrite = dashboard.can_write && !dashboard.archived;
+  const showCaching = canWrite && PLUGIN_CACHING.isGranularCachingEnabled();
+
+  if (page === "caching") {
+    return (
+      <PLUGIN_CACHING.SidebarCacheForm
```

Comment:
I like this simplification. No need for two distinct components. They are very similar.